misc/systemd/cf-consumer.service.in

```diff
@@ -0,0 +1,21 @@
+[Unit]
+Description=CFEngine Enterprise Report Consumer
+PartOf=cfengine3.service
+After=syslog.target
+After=cf-postgres.service
+After=cf-redis-server.service
+ConditionPathExists=@bindir@/cf-consumer
+BindsTo=cf-redis-server.service
+BindsTo=cf-postgres.service
+
+[Service]
+Type=simple
+ExecStart=@bindir@/cf-consumer --no-fork --verbose
```

Comment:
Hey @kacf, when I look at `configure.ac` I do see some entries for `workdir`,
`masterdir`, `piddir`, `inputdir`, `logdir`, and `statedir`. Your right that I
don't see `bindir` anywhere in there.

    AC_SUBST([enable_builtin_extensions])
    AC_SUBST(projlibdir)
    AC_SUBST([CORE_CPPFLAGS])
    AC_SUBST([CORE_CFLAGS])
    AC_SUBST([CORE_LDFLAGS])
    AC_SUBST([CORE_LIBS])
    AC_SUBST(workdir, "${WORKDIR}")
    AC_SUBST(masterdir, "${MASTERDIR}")
    AC_SUBST(inputdir, "${INPUTDIR}")
    AC_SUBST(logdir, "${LOGDIR}")
    AC_SUBST(piddir, "${PIDDIR}")
    AC_SUBST(statedir, "${STATEDIR}")
            AC_SUBST([INIT_D_PATH])
            AC_SUBST([SYSTEMD_SERVICE_PATH])
    AC_SUBST([OS_ENVIRONMENT_PATH])
    AC_SUBST(post_macros)

I did find `sbindir` seems to map to `/var/cfengine/bin`.

    sbindir='${exec_prefix}/bin' # /var/cfengine/bin despite being sbin_?

Would I be able to use `sbindir` instead of `bindir` or should I move to
`@workdir@/bin` as you suggested?

also ping @jimis